_pages/en/strategy-communications.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Communications Strategy
+ref: strategy-communications
+lang: en
+status: posted
+sections: Work In Progress
+permalink: /strategy-communications.html
+---
+
+## Communications Strategy
+
+- [Endorsement](#Endorsement)
+- [Outreach](#outreach)
+- [Feedback](#feedback)
+
+### Endorsement
+
+To be official, strategies, actions and other initiatives, must be endorsed or approved by management (DG, CIO, DM, ..).
+The following are way to get things endorsed.
+
+#### CIOEC Endorsement
+
+Requires wider consultation and presentation to DG Committee.
+Use for quarterly strategies and actions as well as other changes that require wider consultation.
+
+Follow these steps:
+
+1. Director Feedback/updates
+1. Other teams feedback/updates
+   - Security
+   - Operations
+   - Development
+   - Databases
+   - Learning
+   - ..
+1. DG feedback/updates
+1. Present to DG Committee
+1. Present to Architecture Review Committee
+1. Present to CIO Executive Committee
+1. Published on Intranet
+
+#### Briefing Note
+
+Can be sent directly to DG, CIO or DM, using organization reporting structure (hierarchy).
+Use for information or to avoid going up through committees for Endorsement.
+
+### Outreach
+
+Quarterly strategies and actions will start on 2020-04-01.
+Every 3 months the strategies and actions will be updated based on feedback and progress.
+New strategies and actions will be added based on strategy map (layer on the strategies).
+The following are way to get things Outreached.
+
+#### Emails
```

Comment:
To get the CIO to send an e-mail requires the docket process (the same one that we're presently working on demoing the automation of)

So the process is currently kind of defined (at least there is a document that outlines required sign offs, though doesnt define things like whose signature overwrites who and so forth, so it restarts at the beginning each time a modification is required, though I do not know if that is part of the 'official process' or not), though may also be in the process of changing ...

Reply:
This is Outreach section.  How our we reaching out?  Emails, presentations, ...

How we get emails sent should be covered in Endorsement section. (BN or docket process)